workers/requirements.in

```diff
@@ -0,0 +1,9 @@
+amqp
+celery
+django
+requests
+psycopg2
+boto3
+retrying
+boto
```

Comment:
I think I remember asking this before, but why do we need both boto and boto3 again?

Reply:
`boto3` is what we want because it's the latest and greatest, `boto` is used by Celery to communicate with `S3`.

Reply:
Hmm, is there a timeline for when Celery will be compatible with boto3?

Reply:
These docs seem to instruct the user to install `boto3`: http://docs.celeryproject.org/en/master/getting-started/brokers/sqs.html

Reply:
I'm ripping Celery out as part of using Nomad, so I think it's worth leaving it hanging until then.

Reply:
Created https://github.com/data-refinery/data_refinery/issues/73 to make sure this gets removed.

Reply:
:+1: